boards/shields/nxp_btb44_ov5640/doc/index.rst

```diff
@@ -0,0 +1,134 @@
++----------------------+--------------------+
+| 14                   | FREX               |
++----------------------+--------------------+
+| 15                   | HREF               |
++----------------------+--------------------+
+| 16                   | MIPI_CSI_DP1 / D9  |
++----------------------+--------------------+
+| 17                   | PWDN               |
++----------------------+--------------------+
+| 18                   | MIPI_CSI_DN1 / D8  |
++----------------------+--------------------+
+| 19                   | MIPI_CSI_DP1 / D9  |
++----------------------+--------------------+
+| 20                   | DGND               |
++----------------------+--------------------+
+| 21                   | MIPI_CSI_DN1 / D8  |
++----------------------+--------------------+
+| 22                   | MIPI_CSI_CLKP / D7 |
++----------------------+--------------------+
+| 23                   | MIPI_CSI_CLKP / D7 |
++----------------------+--------------------+
+| 24                   | MIPI_CSI_CLKN / D6 |
++----------------------+--------------------+
+| 25                   | MIPI_CSI_CLKN / D6 |
++----------------------+--------------------+
+| 26                   | DGND               |
++----------------------+--------------------+
+| 27                   | MIPI_CSI_DP0 / D5  |
++----------------------+--------------------+
+| 28                   | MIPI_CSI_DP0 / D5  |
++----------------------+--------------------+
+| 29                   | MIPI_CSI_DN0 / D4  |
++----------------------+--------------------+
+| 30                   | MIPI_CSI_DN0 / D4  |
++----------------------+--------------------+
+| 31                   | D3                 |
++----------------------+--------------------+
+| 32                   | DGND               |
++----------------------+--------------------+
+| 33                   | D2                 |
++----------------------+--------------------+
+| 34                   | XCLK               |
++----------------------+--------------------+
+| 35                   | D1                 |
++----------------------+--------------------+
+| 36                   | DVDD               |
++----------------------+--------------------+
+| 37                   | D0                 |
++----------------------+--------------------+
+| 38                   | DOVDD              |
++----------------------+--------------------+
+| 39                   | DGND               |
++----------------------+--------------------+
+| 40                   | DGND               |
++----------------------+--------------------+
+| 41                   | GND                |
++----------------------+--------------------+
+| 42                   | GND                |
++----------------------+--------------------+
+| 43                   | GND                |
++----------------------+--------------------+
+| 44                   | AF_GND             |
++----------------------+--------------------+
```

Comment:
this would probably make sense to also add in the documentation of the new binding for nexus (again, see e.g. the `description` field of `arduino-header-r3` as an example)

Reply:
yes, done